week-5/Homework/mandatory/2-exercises/exercises.js

```diff
@@ -57,7 +75,29 @@ function exerciseTwo(shopping) {
 
     The end result should look something like this: https://hyf-js2-week1-makeme-ex1-demo.herokuapp.com/
 **/
-function exerciseThree(books) {
+function exerciseThree(books , i) {
+  const ListBook = document.getElementById("content");
+  books.forEach(element => {
+    let PEl = document.createElement("p");
+    console.log(element.alreadyRead);
+    const List = document.createElement("ul");
```

Comment:
It's better to use the lower case letters for variable names. e.g. `list` instead of `List`